compiler/luci-micro/helpers/GenerateKernelsListHelper.cpp

```diff
@@ -168,40 +196,33 @@ void run(std::ofstream &os, const circle::Model *model)
       const auto op = ops->Get(i);
       auto op_builtin_operator = reader.builtin_code(op);
 
-      if (operations_set.find(op_builtin_operator) == operations_set.end())
+      auto result = operations_set.insert(op_builtin_operator);
+      if (result.second)
       {
         os << get_register_kernel_str(op_builtin_operator) << std::endl;
-        operations_set.insert(op_builtin_operator);
       }
     }
   }
 }
 
 int main(int argc, char **argv)
 {
-  arser::Arser arser{"Generate KernelBuilder list"};
-  arser.add_argument("circle").help("Circle file to inspect");
-  arser.add_argument("path").help("Path for generated file");
-
-  try
+  if (argc != 3)
   {
-    arser.parse(argc, argv);
-  }
-  catch (const std::runtime_error &err)
-  {
-    std::cout << err.what() << std::endl;
-    std::cout << arser;
-    return 255;
+    throw std::runtime_error(
+      "Should be 2 arguments: circle model path, and path for generated model\n");
   }
 
-  std::string model_file = arser.get<std::string>("circle");
-  std::string generated_file_path = arser.get<std::string>("path");
+  std::string model_file(argv[1]);
+  std::string generated_file_path(argv[2]);
+
+  std::cout << "model: " << model_file << std::endl;
+  std::cout << "file: " << generated_file_path << std::endl;
```

Comment:
Is it intentional or for debug purposes?

Reply:
remove it, thanks :)